src/test/java/uk/gov/ons/census/casesvc/service/CaseReceiptServiceTest.java

```diff
@@ -32,121 +33,210 @@
   @InjectMocks CaseReceiptService underTest;
 
   @Test
-  public void testHHIndReceiptsCase() {
-    // when
-    Case caze = new Case();
-    caze.setCaseType("HH");
-    caze.setAddressLevel("U");
-    caze.setCaseId(UUID.randomUUID());
-    caze.setReceiptReceived(false);
+  public void test_HH_U_HH() {
+    // HH	U	HH	N	Y	Close
```

Comment:
I can see these comments being useful but I think they need a bit more explanation or formatting, I'm just having to guess what each 'column' represents?

Reply:
Arguably the IDE should help you to know what each parameter is. I guess this comment was taken directly from the confluence table?

Is there a case for having a single test, which runs `testReceipting` for all the different combinations, which would make it more clear that it's testing a table of stuff?

Personally, just lose the comments. I'm approving this PR anyway.

Reply:
gone